packages/material-ui-lab/src/DataGrid/DataGrid.js

```diff
@@ -440,18 +522,21 @@ const DataGrid = React.forwardRef(function DataGrid(props, ref) {
                  -        </table>
                  */}
       </div>
-      <table>
+      {paginationState.pagination && <table>
```

Comment:
Maybe?

```suggestion
      {paginationState.pagination ? <table>
```

Reply:
? needs a fallback to be shown of the prop is false. What do you propose as the fallback?

Reply:
Sorry, I should have written down my thoughts. I was considering the ternary vs boolean approach to writing the logic. It's not very important if we can guarantee that `paginationState.pagination` is an actual boolean. https://stackoverflow.com/questions/57961689/rendering-a-component-with-logical-vs-ternary-operator#comment102335617_57962054.

I have taken the habit to always write a ternary. The first few years in my React journey I was && all-in. What's your preference?

Reply:
I see what you mean, since I am using ts, it could never happen to be something different from a boolean/undefined. SO I never encountered that problem. For JS, it might be better to use ?:

Reply:
In our case, TypeScript won't help for values that come from the outside, through the public API.

Reply:
It would if the outside uses typescript as well. Since it would forbid some else due to the types. But our js users could change it. So we should go with the :? Approach.

Reply:
Right, an interesting metric we look at is the TypeScript vs JavaScript usage of the demos. The split is around 10/90% right now, but the trend is toward more TypeScript.

Reply:
That's really interesting. I am sure it will go up, especially because the default of the demos is js, so I always look at the js demos and just transform them for me to ts. I thought it would be more though. What other statistics do you have ;)

Reply:
Yeap, I think that we will see this figure go up :).